OmiseSDK/AdaptableStaticTableViewController.swift

```diff
@@ -1,18 +1,7 @@
 import UIKit
 
 
-#if swift(>=4.2)
 public typealias StaticElementIterable = CaseIterable
```

Comment:
Since `CaseIterable` is a native swift feature we can maybe remove `StaticElementIterable` and just use `CaseIterable` everywhere?

Reply:
Nice catch, removed 🙇